Mastodon/Scene/Onboarding/Login/MastodonLoginView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  MastodonLoginView.swift
+//  Mastodon
+//
+//  Created by Nathan Mattes on 10.11.22.
+//
+
+import UIKit
+import MastodonAsset
+import MastodonLocalization
+
+class MastodonLoginView: UIView {
+
+  // List with (filtered) domains
+
+  let titleLabel: UILabel
+  let subtitleLabel: UILabel
+  private let headerStackView: UIStackView
+  let searchTextField: UITextField
+  let tableView: UITableView
+  private var tableViewWrapper: UIView
+  let navigationActionView: NavigationActionView
+  var bottomConstraint: NSLayoutConstraint?
+
+  override init(frame: CGRect) {
+
+    titleLabel = UILabel()
+    titleLabel.font = MastodonLoginViewController.largeTitleFont
+    titleLabel.textColor = MastodonLoginViewController.largeTitleTextColor
+    titleLabel.text = L10n.Scene.Login.title
+    titleLabel.numberOfLines = 0
+
+    subtitleLabel = UILabel()
+    subtitleLabel.font = MastodonLoginViewController.subTitleFont
+    subtitleLabel.textColor = MastodonLoginViewController.subTitleTextColor
+    subtitleLabel.text = L10n.Scene.Login.subtitle
+    subtitleLabel.numberOfLines = 0
+
+    headerStackView = UIStackView(arrangedSubviews: [titleLabel, subtitleLabel])
+    headerStackView.axis = .vertical
+    headerStackView.spacing = 16
+    headerStackView.translatesAutoresizingMaskIntoConstraints = false
+
+    searchTextField = UITextField()
+    searchTextField.translatesAutoresizingMaskIntoConstraints = false
+    searchTextField.backgroundColor = Asset.Scene.Onboarding.textFieldBackground.color
+    searchTextField.placeholder = L10n.Scene.Login.ServerSearchField.placeholder
+    searchTextField.leftView = UIImageView(image: UIImage(systemName: "magnifyingglass"))
```

Comment:
That's could improve to match the UX design. 

Maybe reference this implementation: https://github.com/mastodon/mastodon-ios/blob/7e04dd3d31e0c71db72ab8752665eaf348c103fe/Mastodon/Scene/Onboarding/PickServer/View/PickServerServerSectionTableHeaderView.swift#L65-L94

Reply:
Awesome, thanks!